test/natalie/regexp_test.rb

```diff
@@ -67,6 +67,12 @@
       r2.should be_kind_of(Regexp)
       r1.should == r2
     end
+
+    it 'automatically applies FIXEDENCODING flag' do
+      c = Regexp.new('\x80', Regexp::NOENCODING)
+      c.options.should == 48
```

Comment:
This `48` looks pretty random, I think it's more readable to use the symbolic names.
```ruby
c.options.should == Regexp::FIXEDENCODING | Regexp::NOENCODING
```